{"url": "https://www.otodom.pl/pl/oferta/mieszkanie-41m2-na-walowej-38-w-centrum-gdanska-ID4ykzj", "error": "Object of type datetime is not JSON serializable", "traceback": "Traceback (most recent call last):\n  File \"D:\\M1A5TO\\realestate-scraper\\scrapper\\pipelines\\detail.py\", line 158, in run_morizon_detail\n    dbg.write(json.dumps({\"url\": u, \"missing\": missing, \"data\": data}, ensure_ascii=False) + \"\\n\")\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 180, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\nTypeError: Object of type datetime is not JSON serializable\n"}
{"url": "https://www.otodom.pl/pl/oferta/gotowe-kawalerki-w-standardzie-premium-4min-do-neptuna-zobacz-sam-ID4yGCD", "error": "Object of type datetime is not JSON serializable", "traceback": "Traceback (most recent call last):\n  File \"D:\\M1A5TO\\realestate-scraper\\scrapper\\pipelines\\detail.py\", line 158, in run_morizon_detail\n    dbg.write(json.dumps({\"url\": u, \"missing\": missing, \"data\": data}, ensure_ascii=False) + \"\\n\")\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 180, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\nTypeError: Object of type datetime is not JSON serializable\n"}
{"url": "https://www.otodom.pl/pl/oferta/kawalerka-na-sprzedaz-gotowa-do-zamieszkania-ID4wAr2", "error": "Object of type datetime is not JSON serializable", "traceback": "Traceback (most recent call last):\n  File \"D:\\M1A5TO\\realestate-scraper\\scrapper\\pipelines\\detail.py\", line 158, in run_morizon_detail\n    dbg.write(json.dumps({\"url\": u, \"missing\": missing, \"data\": data}, ensure_ascii=False) + \"\\n\")\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 180, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\nTypeError: Object of type datetime is not JSON serializable\n"}
{"url": "https://www.otodom.pl/pl/oferta/mieszkanie-w-stanie-deweloperskim-gdansk-aniolki-ID4ys67", "error": "Object of type datetime is not JSON serializable", "traceback": "Traceback (most recent call last):\n  File \"D:\\M1A5TO\\realestate-scraper\\scrapper\\pipelines\\detail.py\", line 158, in run_morizon_detail\n    dbg.write(json.dumps({\"url\": u, \"missing\": missing, \"data\": data}, ensure_ascii=False) + \"\\n\")\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 180, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\nTypeError: Object of type datetime is not JSON serializable\n"}
{"url": "https://www.otodom.pl/pl/oferta/dla-wymagajacego-klienta-120-m2-widok-taras-ID4yL8V", "error": "Object of type datetime is not JSON serializable", "traceback": "Traceback (most recent call last):\n  File \"D:\\M1A5TO\\realestate-scraper\\scrapper\\pipelines\\detail.py\", line 158, in run_morizon_detail\n    dbg.write(json.dumps({\"url\": u, \"missing\": missing, \"data\": data}, ensure_ascii=False) + \"\\n\")\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 180, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\nTypeError: Object of type datetime is not JSON serializable\n"}
{"url": "https://www.otodom.pl/pl/oferta/gotowy-apartament-zobacz-na-zywo-12-min-spacerem-do-wyspy-spichrzow-ID4yGYn", "error": "Object of type datetime is not JSON serializable", "traceback": "Traceback (most recent call last):\n  File \"D:\\M1A5TO\\realestate-scraper\\scrapper\\pipelines\\detail.py\", line 158, in run_morizon_detail\n    dbg.write(json.dumps({\"url\": u, \"missing\": missing, \"data\": data}, ensure_ascii=False) + \"\\n\")\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"D:\\pycharn\\Lib\\json\\encoder.py\", line 180, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\nTypeError: Object of type datetime is not JSON serializable\n"}
